README.md

```diff
@@ -139,6 +139,7 @@ Currently supported platforms:
 * Windows
 * iOS
 * macOS
+* Wine (version 7.13+, see [issue #1444])
```

Comment:
Let's revert these changes as we still have a number of test failures.

Reply:
Reverted doc changes and added note about cfg(wine) to CI config. (https://github.com/tokio-rs/mio/pull/1637/commits/16f6789b00edfb54ec215012be2ab0fdb86a6da5)